Mathlib/Topology/PathConnected.lean

```diff
@@ -326,11 +324,8 @@ def trans (γ : Path x y) (γ' : Path y z) : Path x z where
     refine'
       (Continuous.if_le _ _ continuous_id continuous_const (by norm_num)).comp
         continuous_subtype_val <;>
-    -- TODO: the following are provable by `continuity` but it is too slow
-    -- porting note: the new `continuity` succeeds and it isn't too slow!
+    -- Porting note: used `continuity`
```

Comment:
```suggestion
```
I think just drop this, it's confusing without the context, and doesn't really need to be recorded.